### PR TITLE
Upgrade to Rector v1

### PR DESCRIPTION
**Pull request by deluxetom**
I tested it in a personal hyperf project and it worked well

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,16 +5,12 @@
...
         "php": ">=8.0",
-        "rector/rector": "^0.15.12"
+        "rector/rector": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^10.0",
         "phpstan/phpstan": "^1.8.2",
         "symplify/phpstan-rules": "^11.0",
-        "symplify/phpstan-extensions": "^11.0",
-        "symplify/easy-coding-standard": "^11.0",
```
Comment (huangdijia) [inline composer.json:15]: required when execute `composer fix-cs`

Comment (deluxetom) [reply composer.json:15]: removed that one by accident, I put it back

**Comment by huangdijia** [comment]
Actions ci did not pass

**Comment by deluxetom** [comment]
thanks for the fixes and the merge!

**Pull request merged**